Library/Homebrew/utils/analytics.rb

```diff
@@ -313,8 +313,12 @@ def default_tags_influx
       sig { returns(T::Hash[Symbol, String]) }
       def default_fields_influx
         @default_fields_influx ||= begin
-          version = HOMEBREW_VERSION.match(/^[\d.]+/)[0]
-          version = "#{version}-dev" if HOMEBREW_VERSION.include?("-")
+          version = if HOMEBREW_VERSION.start_with?(">=")
+            "unknown"
```

Comment:
```suggestion
            ">=4.1.21"
```

so we can bump this periodically. Might even be worth passing through the brew.sh value through as a environment variable so they can be shared here.